src/index.js

```diff
@@ -0,0 +1,10 @@
+// this is the entry point for the library
+// it exports all the functions that are available to the user
+// it uses also for the bundling and minifying
+// (see package.json's "build" script)
+import { jUnit } from './junit.js';
+import { humanizeValue, textSummary } from './text.js';
+
+exports.jUnit = jUnit;
+exports.humanizeValue = humanizeValue;
+exports.textSummary = textSummary;
```

Comment:
```suggestion
export { jUnit } from './junit.js';
export { humanizeValue, textSummary } from './text.js';
```

1. this is ... more ESM-y
2. mixing commonjs and ESM is not good idea and soon will not be supported in k6 ... or at least not in a good way.